vignettes/palettes.Rmd

```diff
@@ -0,0 +1,291 @@
+show_palette(traffic_palette(3))
+```
+
+```{r traffic-palette-3-demo, fig.cap = "`traffic_palette()` with 3 levels."}
+demo_palette(traffic_palette(3))
+```
+
+\FloatBarrier
+
+## `ordinal_palette()`
+
+`ordinal_palette()` has a variable number of colours.
+It starts with a dark colour and end with a light colour.
+Other colour are intermediate colours, resulting is a gradient from dark to light.
+The colours depend on the active `ggplot2` theme.
+
+```{r ordinal-palette-inbo, fig.height = show_height, fig.cap = "`ordinal_palette(5)` for `theme_inbo()`."}
+theme_set(theme_inbo())
+show_palette(ordinal_palette(5))
+```
+
+```{r ordinal-palette-inbo-demo, fig.cap = "`ordinal_palette(5)` with `theme_inbo()`."}
+demo_palette(ordinal_palette(5))
+```
+
+```{r ordinal-palette-vl, fig.height = show_height, fig.cap = "`ordinal_palette(5)` for `theme_vlaanderen2015()`."}
+theme_set(theme_vlaanderen2015())
+show_palette(ordinal_palette(5))
+```
+
+```{r ordinal-palette-vl-demo, fig.cap = "`ordinal_palette(5)` with `theme_vlaanderen2015()`."}
+demo_palette(ordinal_palette(5))
+```
+
+```{r ordinal-palette-elsevier, fig.height = show_height, fig.cap = "`ordinal_palette(5)` for `theme_elsevier()`."}
+theme_set(theme_elsevier())
+show_palette(ordinal_palette(5))
+```
+
+```{r ordinal-palette-elsevier-demo, fig.cap = "`ordinal_palette(5)` with `theme_elsevier()`."}
+demo_palette(ordinal_palette(5))
+```
+
+\FloatBarrier
+
+# Named colours
+
+## INBO colours
+
+```{r inbo-named, fig.height = show_height, echo = TRUE}
+inbo_named <- c(
+  inbo_donkerblauw, inbo_munt, inbo_oranjerood, inbo_grijsblauw,
+  inbo_lichtgroen, inbo_groen, inbo_lichtgrijs, inbo_kaki, inbo_felrood,
+  inbo_steun_blauw, inbo_geelgr, inbo_steun_geelgroen, inbo_steun_donkerroos,
+  inbo_donkergroen, inbo_oranje, inbo_geel, inbo_lichtblauw, inbo_hoofd,
+  inbo_rood, inbo_grijs, inbo_bruinrood, inbo_oranje, inbo_groen
+)
+show_palette(inbo_named)
+```
+
+```{r inbo-named-demo}
+demo_palette(inbo_named)
+```
+
+## Flanders colours
+
+```{r vl-named, fig.height = show_height, echo = TRUE}
+vl_named <- c(
+  vl_black, vl_darkblue, vl_darkbrown, vl_darkgreen, vl_darkred, vl_darkyellow,
+  vl_grey1, vl_grey2, vl_grey3, vl_grey4, vl_grey5, vl_grey6, vl_grey7,
+  vl_lightblue, vl_lightbrown, vl_lightgreen, vl_lightred, vl_yellow
+)
+show_palette(vl_named)
+```
+
+```{r vl-named-demo}
+demo_palette(vl_named)
+```
+
+# Simulate effect of colour blindness on the perception of palettes
+
+## `inbo_palette()`
+
+```{r inbo-palette-2, fig.height = show_height}
+show_palette(inbo_palette())
+```
+
+```{r inbo-cluster, echo = FALSE, fig.cap = "Agglomerative hierarchical clustering of colours based on their distance."}
+library(cluster)
+inbo_palette() |>
+  colour_blind_distance() |>
+  agnes(method = "complete") |>
+  plot(ask = FALSE, which.plots = 2, main = "")
+```
+
+```{r inbo-palette-deutan, fig.cap = "Simulation of deutan model of colorblindness."}
```

Comment:
colorblindness is written as two words in the title but as 1 word in the figure captions. I think both can be used but bette rnot to use them both in the same text. "color blindness" has more hits on google than "colorblindness"